microsoft-edge/devtools-guide-chromium/3d-view/index.md

```diff
@@ -16,8 +16,6 @@ Use the **3D View** to debug your web app by navigating through the [Document Ob
 *   [Clear some of the clutter on the DOM pane](#changing-your-view) or the [z-index pane](#change-the-scope-of-your-exploration).
 *   [Pick the color scheme to best debug your DOM problems](#dom-color-type) or [z-index problems](#z-index-color-type).
 
-To explore an early prototype for the 3D View project and run the code yourself, go to [3D View Sample](https://github.com/MicrosoftEdge/DevToolsSamples/tree/master/3DView).
```

Comment:
For info, I removed this part entirely. This does not help our readers learn to use the 3D view at all. This might have been important a few years ago when the tool was first being built, but I don't see the point anymore.